src/views/App/index.tsx

```diff
@@ -30,6 +30,7 @@ class App extends React.Component<MainProps, MainState> {
               <SecureRoute
                 exact
                 path="/system/:systemId/grt-review"
+                render={({ component }: any) => component()}
```

Comment:
Can we get a better type and `any` here? I think React has an exported type for component.